releasenotes/notes/add-copy-url-b39441ee986bf333.yaml

```diff
@@ -0,0 +1,2 @@
+features:
+  - add a new option in the context menu (copy url)
```

Comment:
cool. no need to split these into 3 files

Reply:
Sure, sorry about that! 🙂